modules/manage/pages/kubernetes/k-schema-controller.adoc

```diff
@@ -0,0 +1,272 @@
+= Manage Schemas with the Redpanda Operator
+:description: Use the Schema resource to declaratively create and manage schemas as part of a Redpanda deployment in Kubernetes.
+:page-categories: Management, Development
+:env-kubernetes: true
+
+Use the Schema resource to declaratively create and manage schemas as part of a Redpanda deployment in Kubernetes. Each Schema resource maps to a schema in your Redpanda cluster, allowing you to define data structures, compatibility, and schema evolution in a declarative way.
+
+== Prerequisites
+
+Ensure you have the following:
+
+* *Kubectl*: Ensure you have the https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl^] command-line tool installed and configured to communicate with your cluster.
+* *Redpanda cluster*: Ensure you have at least version v2.3.0-24.2.10 of the xref:deploy:deployment-option/self-hosted/kubernetes/k-production-deployment.adoc[Redpanda Operator] and a Redpanda resource deployed and accessible.
+
+== Create a schema
+
+. Define a schema using the Schema resource. Here's a basic example configuration that defines an Avro schema:
++
+.`schema.yaml`
+[source,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha2
+kind: Schema
+metadata:
+  name: example-schema
+  namespace: <namespace>
+spec:
+  cluster:
+    clusterRef:
+      name: <cluster-name>
+  schemaType: avro
+  compatibilityLevel: Backward
+  text: |
+    {
+      "type": "record",
+      "name": "ExampleRecord",
+      "fields": [
+        { "type": "string", "name": "field1" },
+        { "type": "int", "name": "field2" }
+      ]
+    }
+----
++
+Replace the following placeholders:
++
+- `<namespace>`: The namespace in which to deploy the Schema resource. The Schema resource must be deployed in the same namespace as the Redpanda resource defined in `clusterRef.name`.
+- `<cluster-name>`: The name of the Redpanda resource that defines the Redpanda cluster to which you want to upload the schema.
+
+. Apply the manifest:
++
+[source,bash]
+----
+kubectl apply -f schema.yaml --namespace <namespace>
+----
++
+When the manifest is applied, the schema will be created in your Redpanda cluster.
+
+. Check the status of the Schema resource using the following command:
++
+[,bash]
+----
+kubect get schema example-schema --namespace <namespace>
+----
+
+. Verify that the schema was created in Redpanda:
++
+[source,bash]
+----
+kubectl exec -it --namespace <namespace> <pod-name> -- curl https://<pod-name>.<cluster-name>.<namespace>.svc.cluster.local:8081/subjects -sS --cacert /etc/tls/certs/default/ca.crt -w '\n'
+----
++
+Replace `<pod-name>` with the name of a Pod that's running Redpanda.
+
+== Schema examples
+
+These examples demonstrate how to define schemas in Avro, Protobuf, and JSON Schema formats.
```

Comment:
(This probably mostly applies to me, so note to myself:) We should standardize when referring to the schema format - "JSON Schema" is more specific than just "JSON" 👍